jsonschema/_validators.py

```diff
@@ -496,9 +501,6 @@ def if_(validator, if_schema, instance, schema):
             yield error
 
 
-
-
-
 def unevaluatedItems(validator, unevaluatedItems, instance, schema):
```

Comment:
Belongs in another commit. Though, it's up to @Julian on how clean the commits are.

Reply:
We're going to have to redo some of these commits anyhow to get the tests upstream into the test suite -- @nezhar maybe it's a decent time to start doing some of that? Otherwise they'll start to pile up.

Overall though I don't mind much if the commits aren't 100% clean, just keep them as clean as you can and if I can't decipher them I'll ask.

Reply:
Currenlty all tests are fetched form the external test suite via https://github.com/Julian/jsonschema/pull/817/commits/d371dc53007ee947e81e049d2624ff46a8674551 - I can move this commit up in the history or create a dedicated PR so it does not polute the list of changes.

Would it make sense to squash the commits the implementation is done? I was thinking to keep them like this for the review, but certain keywords need to be extended when implementing others.